tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I wonder what is the tradeoff if we aim for a higher target, like `es6`.

Reply:
I added new target now 👌